Dixit_Logic/Interfaces/IDixitGame.cs

```diff
@@ -12,58 +12,23 @@ namespace Dixit_Logic.Interfaces
     interface IDixitGame
     {
         /// <summary>
-        /// It contain all players who playe in the acutal game. 
+        /// This holds the actual game state what is changed by actions during the game.
         /// </summary>
-        IList<IPlayer> Players
+        IGameState ActualGameState
         {
             get;
             set;
         }
 
         /// <summary>
-        /// This is the text what describe the association of turn's starter card.
-        /// </summary>
-        String CardAssociationText
-        {
-            get;
-            set;
-        }
-
-        /// <summary>
-        /// The player who start the actual turn.
-        /// </summary>
-        IPlayer ActualPlayer
-        {
-            get;
-        }
-
-        /// <summary>
-        /// This is the deck from wich the players draw new cards.
-        /// </summary>
-        IDeck MainDeck
-        {
-            get;
-            set;
-        }
-
-        /// <summary>
-        /// This are the cards what the players put on the table.
-        /// </summary>
-        IDeck BoardDeck
-        {
-            get;
-            set;
-        }
-
-        /// <summary>
-        /// This add a new player (by name) to the Players.
+        /// This add a new player (by name) to the ActualGameState.
         /// </summary>
         /// <param name="name">The name of the player.</param>
         /// <returns>The new player identifier</returns>
         int AddPlayer(String name);
```

Comment:
Also in the PutCard method.. It is not an int that is putting the card, but a Player. Makes the code more readable.